include/email_challenge.hpp

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "user_handle.hpp"
+
+namespace emailchallenge {
+
+bool EmailChallenge(bool check_input_email, const std::string &input_email, const userhandle::UserHandle &user, const int y, const std::string &prompt, const std::string &ip, const std::string &filename, int *out_y);
```

Comment:
Could you wrap the line at 80 or 100 characters, and align the following lines with the open parentheses?